internal-dns/src/config.rs

```diff
@@ -407,6 +407,27 @@ impl DnsConfigBuilder {
             (name, vec![DnsRecord::Aaaa(sled_ip)])
         });
 
+        // Assemble the special boundary NTP name to support chrony on internal
```

Comment:
Worth adding a unit test for this?

Reply:
I'm on the fence - the expanded `test_blueprint_internal_dns_basic` does check for this name. If you'd like a unit test specific to this crate, I can add one?

Reply:
Sure enough! Added in [24b7562](https://github.com/oxidecomputer/omicron/pull/6050/commits/24b7562795769a6cde1de2e045f35f9c64ff968e)